server.local.js

```diff
@@ -1,5 +1,5 @@
 const setup = require('./server')
-const port = 3000
```

Comment:
Why the port change?

Reply:
![image](https://user-images.githubusercontent.com/53128285/167537224-4c112ac3-e1cc-455b-9972-17a4493666da.png)

Reply:
I don't remember why it was 3000 initially, I probably just wasn't paying attention and never ran the backend locally anwyays.